debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -116,48 +116,44 @@ const djdt = {
             const toggleClose = "-";
             const openMe = this.textContent === toggleOpen;
             const name = this.dataset.toggleName;
-            const container = document.getElementById(name + "_" + id);
-            container
-                .querySelectorAll(".djDebugCollapsed")
-                .forEach(function (e) {
-                    $$.toggle(e, openMe);
-                });
-            container
-                .querySelectorAll(".djDebugUncollapsed")
-                .forEach(function (e) {
-                    $$.toggle(e, !openMe);
-                });
-            const self = this;
+            const container = document.getElementById(`${name}_${id}`);
+            container.querySelectorAll(".djDebugCollapsed").forEach((e) => {
+                $$.toggle(e, openMe);
+            });
+            container.querySelectorAll(".djDebugUncollapsed").forEach((e) => {
+                $$.toggle(e, !openMe);
+            });
             this.closest(".djDebugPanelContent")
-                .querySelectorAll(".djToggleDetails_" + id)
-                .forEach(function (e) {
+                .querySelectorAll(`.djToggleDetails_${id}`)
+                .forEach((e) => {
                     if (openMe) {
                         e.classList.add("djSelected");
                         e.classList.remove("djUnselected");
-                        self.textContent = toggleClose;
+                        this.textContent = toggleClose;
```

Comment:
Good catch

Reply:
Not a bugfix by the way, just something enabled by the use of arrow functions instead of function() {}.